scene/resources/default_theme/default_theme.cpp

```diff
@@ -570,7 +570,6 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	// Window
 
 	theme->set_stylebox("embedded_border", "Window", sb_expand(make_flat_stylebox(style_popup_color, 10, 28, 10, 8), 8, 32, 8, 6));
-	theme->set_constant("scaleborder_size", "Window", 4 * scale);
```

Comment:
What exactly is this change and what does it do, or rather, why in this PR? :) A bit confusing to me in a docs PR.